session6/src/homework/ForLoopEx.java

```diff
@@ -0,0 +1,27 @@
+package homework;
+
+public class ForLoopEx {
+    public static void main(String[] args) {
+
+        //PRINT 1 TO 10 USING FOR
+
+        //System.out.println("Output is : ");
+
+        //for(int i = 1; i <= 10; i++)
+        {
+            //System.out.println(i);
```

Comment:
don't comment the code, you can have both `for` loops in this main, they don't impact each other.